modules/vault-elb/outputs.tf

```diff
@@ -15,5 +15,5 @@ output "load_balancer_security_group_id" {
 }
 
 output "fully_qualified_domain_name" {
-  value = "${aws_route53_record.vault_elb.fqdn}"
+  value = "${aws_route53_record.vault_elb.*.fqdn}"
```

Comment:
This will turn this output into a list, which is probably not what we want. You instead need to use the slightly ugly workaround [listed here](https://www.terraform.io/upgrade-guides/0-11.html#referencing-attributes-from-resources-with-count-0):

```hcl
value = "${element(concat(aws_route53_record.vault_elb.*.fqdn, list("")), 0)}"
```

Reply:
Good point. Wasn't aware of that guide. Have a look.